internal/compose/compose.go

```diff
@@ -179,10 +179,13 @@ func NewProject(name string, paths ...string) (*Project, error) {
 		c.dockerComposeV1 = true
 		return &c, nil
 	}
+
+	versionMessage := fmt.Sprintf("Determined Docker Compose version: %v", ver)
 	if ver.Major() == 1 {
-		logger.Debugf("Determined Docker Compose version: %v, the tool will use Compose V1", ver)
+		versionMessage = fmt.Sprintf("%s, the tool will use Compose V1", versionMessage)
 		c.dockerComposeV1 = true
 	}
+	logger.Debug(versionMessage)
```

Comment:
Maybe it can be dismissed `the tool will use Compose V1` sentence and just lave the message `Determined Docker Compose version: %v`?

Reply:
Though well, it also tells if V1 implementation is being used here.

Reply:
Ok, I'll leave it adding that sentence if docker-compose version is v1.x